src/test/java/de/retest/suite/flow/ApplyChangesToStatesFlowTest.java

```diff
@@ -36,11 +36,12 @@
 	public TemporaryFolder temp = new TemporaryFolder();
 
 	@Rule
-	public SystemProperty recheckDirProp = new SystemProperty( ProjectConfiguration.RETEST_PROJECT_ROOT );
+	public final ProvideSystemProperty recheckDirProp =
```

Comment:
Hmmm, according to the [System Rules Javadoc](https://stefanbirkner.github.io/system-rules/apidocs/org/junit/contrib/java/lang/system/ProvideSystemProperty.html):

>If you want to set a property for a single test then you can use [`RestoreSystemProperties`](https://stefanbirkner.github.io/system-rules/apidocs/org/junit/contrib/java/lang/system/RestoreSystemProperties.html) along with `System.setProperty(String, String)`.